src/main/kotlin/exercise2/task2/FindPairOfHighestSumFunctional.kt

```diff
@@ -20,7 +20,10 @@ import org.jetbrains.exercise2.task3.findPairWithBiggestDifference
  */
 
 internal fun List<Int>.findHighestSumPairFunctional(): Pair<Int, Int> {
-    TODO("Implement me!!")
+//    var par = Pair<Int, Int>(0, 0)
```

Comment:
Please clean up all the comments before pushing the changes and creating the PR.